ocs_ci/deployment/mce.py

```diff
@@ -0,0 +1,243 @@
+
+        # Wait for catalog source is ready
+        mce_catalog_source.wait_for_state("READY")
+
+    def create_mce_namespace(self):
+        """
+        Creates the namespace for mce resources
+
+        Raises:
+            CommandFailed: If the 'oc create' command fails.
+        """
+        try:
+            logger.info(f"Creating namespace {self.namespace} for mce resources")
+            namespace_yaml_file = templating.load_yaml(constants.MCE_NAMESPACE_YAML)
+            namespace_yaml = OCS(**namespace_yaml_file)
+            namespace_yaml.create()
+            logger.info(f"MCE namespace {self.namespace} was created successfully")
+        except exceptions.CommandFailed as ef:
+            if (
+                f'project.project.openshift.io "{self.namespace}" already exists'
+                in str(ef)
+            ):
+                logger.info(f"Namespace {self.namespace} already present")
+                raise ef
+
+    def create_multiclusterengine_operator(self):
+        """
+        Creates multiclusterengine operator
+
+        """
+        operatorgroup_yaml_file = templating.load_yaml(constants.MCE_OPERATOR_YAM)
+        operatorgroup_yaml = OCS(**operatorgroup_yaml_file)
+        try:
+            operatorgroup_yaml.create()
+            logger.info("mce OperatorGroup created successfully")
+        except exceptions.CommandFailed as ef:
+            if "multiclusterengine exists" in str(ef):
+                logger.info("multiclusterengine already exists")
+
+        cmd = "oc get mce multiclusterengine -o jsonpath='{.status}'"
+        cmd_res = exec_cmd(cmd, shell=True)
+        if cmd_res.returncode != 0:
+            logger.error(f"Failed to get multicluster engine status\n{cmd_res.stderr}")
+        else:
+            logger.info(
+                f"Multicluster engine version: {cmd_res.stdout.decode('utf-8')}"
+            )
+            assert (
+                cmd_res.stdout.decode("utf-8") == "Available"
+            ), "multiclusterengine is not is 'Available' status"
+
+    def create_mce_subscription(self):
+        """
+        Creates subscription for mce operator
+
+        """
+        mce_subscription_yaml_data = templating.load_yaml(
+            constants.MCE_SUBSCRIPTION_YAML
+        )
+
+        if config.DEPLOYMENT.get("mce_latest_stable"):
+            mce_subscription_yaml_data["spec"][
+                "source"
+            ] = constants.OPERATOR_CATALOG_SOURCE_NAME
+            mce_sub_channel = "stable-2.7"
+
+        mce_subscription_yaml_data["spec"]["channel"] = f"{mce_sub_channel}"
+        mce_subscription_manifest = tempfile.NamedTemporaryFile(
+            mode="w+", prefix="mce_subscription_manifest", delete=False
+        )
+        templating.dump_data_to_temp_yaml(
+            mce_subscription_yaml_data, mce_subscription_manifest.name
+        )
+        logger.info("Creating subscription for mce operator")
+        run_cmd(f"oc create -f {mce_subscription_manifest.name}")
```

Comment:
please check if subscription exists in the beginning of the method; if exists skip execution of the method